src/Compilers/CSharp/Portable/Symbols/Tuples/TupleErrorFieldSymbol.cs

```diff
@@ -177,5 +177,24 @@ public bool Equals(TupleErrorFieldSymbol other, TypeCompareKind compareKind)
                 _tupleElementIndex == other._tupleElementIndex &&
                 TypeSymbol.Equals(ContainingType, other.ContainingType, compareKind);
         }
+
+        internal override FieldSymbol AsMember(NamedTypeSymbol newOwner)
```

Comment:
>AsMember [](start = 38, length = 8)

I think we want targeted tests for this implementation to cover breadth of possible scenarios, including long tuples, etc. We should verify internal state (all the fields) of the symbol we get back.